tools/govulncheck-parser/main_test.go

```
@@ -0,0 +1,116 @@
+)
+
+func TestCheckCVE(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		cveID    string
+		expected string
+	}{
+		{
+			name:     "affected - CVE has matching OSV and finding",
+			cveID:    "CVE-2025-22868",
```

golangci-lint (2.12.2)

[error] string `CVE-2025-22868` has 9 occurrences, make it a constant

(goconst)